doc/code_snippets/snippets/sharding/templates/basic/{{.name}}-scm-1.rockspec.tt.template

```diff
@@ -0,0 +1,12 @@
+package = '{{ .name }}'
+version = 'scm-1'
+source  = {
+    url = '/dev/null',
+}
+
+dependencies = {
+    'vshard == 0.1.25'
```

Comment:
As I can see, the latest `vshard` version is 0.1.24. Suppose, `tt rocks make` will not work with this rockspec until 0.1.25 is out.

Reply:
@Totktonada Am I correct that the latest config should work with vshard 0.1.25 only? If the new version will be release at the same time as 3.0.0, keeping the new vshard option in the doc sample should be OK. Otherwise, don't know which is the best decision.

Reply:
UPD: see also https://github.com/tarantool/tarantool/pull/9533/files

Reply:
Vshard was released today. It was carefully planned :)